opal/tests/test_core_search_queries.py

```diff
@@ -187,6 +187,28 @@ def test_episodes_for_date_fields_after(self):
     def test_episodes_for_m2m_fields(self):
         criteria = dict(
             column='hat_wearer', field='Hats',
+            combine='and', query='Derby', queryType='Equals'
```

Comment:
Aren't these the wrong way round ? e.g. the test for synonym is called for_m2m and the test for m2m synonyms is m2m_fields_with_synonyms ?